Amendment to remove mention of derivative catalogs (#20)
lsst-dm/dmtn-199: 4eliminate.tex

will be determined by comparison with an appropriate catalog or catalogs at the \gls{USDF}.
Any object that corresponds to an object in this catalog will not be released in the  prompt
alerts or stored in the prompt alert database.

> SLAC will not create any catalogs noting these corresponding notifications.

Reviewer: Not sure this now flows correctly .. there are no corresponding notifications.

Author: I agree, flow is not great...

> SLAC will not create any catalogs noting these corresponding notifications.

Reviewer: ```suggestion
SLAC will not create any catalogs with information corresponding to these object.
```

Reviewer 2: Wil's amendment is fine by me (with fix that it should be "objects").

Author: Agreed, that's better. I'll accept and do Richard's adjustment too.

> SLAC will not create any catalogs noting these corresponding notifications.

Reviewer 3: Is "SLAC" the cognizant party here?

Author: You can see from this that is the party that DOE holds responsible...

> SLAC will not create any catalogs with information corresponding to these object.

Author: ```suggestion
SLAC will not create any catalogs with information corresponding to these objects.
```
